cmd/eks-a-tool/cmd/vspherelogoutsessions.go

```diff
@@ -11,7 +11,6 @@ import (
 	"strings"
```

Comment:
Why are you modifying vsphere code?

Reply:
we're extracting the function used here to reuse it. Without it, the dry-run flag wasn't binding correctly and it would always show as "false" even when I passed it in.